system/Language/Language.php

```diff
@@ -152,7 +152,7 @@ protected function parseLine(string $line): array
 		$file = substr($line, 0, strpos($line, '.'));
 		$line = substr($line, strlen($file) + 1);
 
-		if (! array_key_exists( $line, $this->language[$this->locale][$file] ) )
+		if (! isset($this->language[$this->locale][$file]) || ! array_key_exists($line, $this->language[$this->locale][$file]))
```

Comment:
Why are you checking both ways? All that should be needed is array_key_exists. It's the same as isset, except it works even if the value is empty.

Reply:
e.g "SomeName.en" would array_key_exists( $line, $this->language ) result in true if there is any english file loaded

Reply:
Oh, yes, totally mis-read that Sorry about that!